tests/platform_tests/daemon/test_pcied.py

```diff
@@ -154,26 +121,27 @@ def test_pmon_pcied_term_and_start_status(check_daemon_status, duthosts, rand_on
     @summary: This test case is to check the pcied terminated and restarted status
     """
     duthost = duthosts[rand_one_dut_hostname]
+
+    if "201811" in duthost.os_version or "201911" in duthost.os_version:
+        pytest.skip("Skip: SIG_TERM behaves differnetly in {} on {}".format(daemon_name, duthost.os_version))
+
     pre_daemon_status, pre_daemon_pid = duthost.get_pmon_daemon_status(daemon_name)
     logger.info("{} daemon is {} with pid {}".format(daemon_name, pre_daemon_status, pre_daemon_pid))
 
     duthost.stop_pmon_daemon(daemon_name, SIG_TERM, pre_daemon_pid)
-    time.sleep(2)
 
     daemon_status, daemon_pid = duthost.get_pmon_daemon_status(daemon_name)
-    pytest_assert(daemon_status == expected_exited_status,
-                          "Pcied expected terminated status is {} but is {}".format(expected_exited_status, daemon_status))
-    pytest_assert(daemon_pid == -1,
-                          "Pcied expected pid is -1 but is {}".format(daemon_pid))
+    logger.info("{} daemon got killed unexpectedly and it is in {} with pid {}".format(daemon_name, daemon_status, daemon_pid))
```

Comment:
Why are you logging the info without checking the status if the daemon actually killed or not?

Reply:
The daemon_status check is followed right after but this is just added for debugging purpose. I can remove this part.